test/e2e/install/install_test.go

```diff
@@ -201,9 +201,14 @@ var _ = Describe("[Install] durability", func() {
 			err := e2eclient.Client.Get(context.TODO(), nname, nroObj)
 			Expect(err).ToNot(HaveOccurred())
 
+			By("waiting for the DaemonSet to be created..")
 			uid := nroObj.GetUID()
-			ds, err := getDaemonSetByOwnerReference(uid)
-			Expect(err).ToNot(HaveOccurred())
+			ds := &v1.DaemonSet{}
+			Eventually(func() error {
+				var err error
+				ds, err = getDaemonSetByOwnerReference(uid)
+				return err
+			}, 5*time.Minute, 10*time.Second).Should(BeNil())
```

Comment:
this is a good change we should merge anyway. Better to split on a separate PR we can merge quickly

Reply:
The problem I see with splitting this in another PR is that if this test is not being executed we cannot be sure it is really ok

Reply:
The test is executed under the ```ci-e2e-install``` lane.
Why do you think it's not?